flang/runtime/command.cpp

```diff
@@ -125,6 +126,33 @@ static bool FitsInDescriptor(
       kind, terminator, value);
 }
 
+void removeNewLine(char *str) {
+  char *newlinePos = strchr(str, '\n');
+  if (newlinePos != NULL) {
```

Comment:
Use nullptr instead of NULL.
Alternatively in this case you can just do:
`if (newlinePos) {`. This is idiomatic in C/C++ for checking pointer validity